itest/tarod_harness.go

```diff
@@ -114,11 +116,15 @@ func newTarodHarness(ht *harnessTest, cfg tarodConfig) (*tarodHarness, error) {
 		return nil, err
 	}
 
-	// We'll modify the config slightly here, since we don't need to use
-	// the hashmail system for integration tests.
+	// Conditionally avoid setting up the hashmail system. Many
+	// tests don't require the hashmail system.
+	//
+	// We can skip setting up the hashmail system by modifying the config.
 	//
 	// TODO(roasbeef): make local aperture instance in future
-	finalCfg.HashMailAddr = ""
+	if !enableHashMail {
```

Comment:
Can remove the `TODO` above now.

Reply:
Removed in a later commit within this PR: https://github.com/lightninglabs/taro/pull/248/commits/ceb0c18c7078b7e2e9004a7d35a9a2e3b00522db#diff-5f0a426a3963099e25960fe974827293e079126d4ab5f436335cd7c6bbb9a8b9